core/src/main/java/hudson/Proc.java

```diff
@@ -231,7 +231,7 @@ private static ProcessBuilder environment(ProcessBuilder pb, String[] env) {
             return pb;
         }
 
-        private LocalProc( String name, ProcessBuilder procBuilder, InputStream in, OutputStream out, OutputStream err ) throws IOException {
+        private LocalProc( String name, ProcessBuilder procBuilder, InputStream in, OutputStream out, OutputStream err) throws IOException {
```

Comment:
I'm not a normal contributor to this project so it's probably not my place to nitpick, but all of the changes in this file appear to be purely cosmetic--did you mean to commit these?